bears/xml2/XMLBear.py

```diff
@@ -40,21 +54,30 @@ class XMLBear:
     _output_regex = re.compile(
         r'.*:(?P<line>\d+):.*(?P<severity>error|warning)\s?: '
         r'(?P<message>.*)\n.*\n.*')
+    _diff_severity = RESULT_SEVERITY.INFO
+    _styles = ('c14n', 'c14n11', 'exc-c14n', 'oldxml10')
 
-    @staticmethod
-    def create_arguments(filename, file, config_file,
+    def create_arguments(self, filename, file, config_file,
                          xml_schema: path='',
-                         xml_dtd: path_or_url=''):
+                         xml_dtd: path_or_url='',
+                         xml_style: xml_style_valid=None):
         """
         :param xml_schema: ``W3C XML Schema`` file used for validation.
         :param xml_dtd:    ``Document type Definition (DTD)`` file or
                            url used for validation.
+        :param xml_style:  ``XML Style Specification`` Relevant args are
+                           c14n, c14n11, exc-c14n and oldxml10.
+                           Find out more about the formats at
+                           https://www.w3.org/TR/#tr_XML_Canonicalization
```

Comment:
can you rather give a link where I can *immediately* see what those things mean for my code?  This is not helpful (maybe a bit) but almost nobody will actually browse that website since it doesn't seem to contain the desired info easily available.